lms/djangoapps/certificates/api.py

```diff
@@ -34,6 +34,15 @@
 log = logging.getLogger("edx.certificate")
 
 
+def is_passing_status(cert_status):
```

Comment:
i deliberately added this so that we wouldn't be importing models in other packages, and instead use the api.py functions.   this is consistent with the current usage/intent of the api.py module.